arangod/VocBase/Methods/Databases.cpp

```diff
@@ -544,7 +544,14 @@ Result Databases::drop(ExecContext const& exec, TRI_vocbase_t* systemVocbase,
     auto cb = [&](auth::User& entry) -> bool {
       return entry.removeDatabase(dbName);
     };
-    res = um->enumerateUsers(cb, /*retryOnConflict*/ true);
+    if (auto cleanupUsersRes = um->enumerateUsers(cb, /*retryOnConflict*/ true);
+        cleanupUsersRes.fail()) {
+      LOG_TOPIC("9f8b7", ERR, Logger::AUTHORIZATION)
```

Comment:
If this error is somewhat expected, I think we should log it only with the WARNING log level.

Reply:
It's not expected, that is quite a special case. It only occurs when the coordinator cannot contact the leader of the `_users` collection, in order to perform the transaction on it. That should very seldom be the case, especially now with the fix in fuerte.

Reply:
Back then we decided that if you want the customer to immediately look at the deployment, create an error. If it's something for the next working day, use a warning. In that case, I would say a warning is sufficient.

Reply:
I changed the log level to `WARN` for this PR and updated the back-ports accordingly.